forest/tests/config_tests.rs

```diff
@@ -121,3 +122,60 @@ fn test_config_env_var() {
 
     assert!(expected_config == actual_config);
 }
+
+#[test]
+fn test_download_location_of_proof_parameter_files_env() {
+    let config = Config::default();
+
+    let mut config_file = tempfile::Builder::new().tempfile().unwrap();
+    config_file
+        .write_all(toml::to_string(&config).unwrap().as_bytes())
+        .expect("Failed writing configuration!");
+
+    let tmp_dir = TempDir::new().unwrap();
+
+    Command::cargo_bin("forest")
+        .unwrap()
+        .env("FOREST_CONFIG_PATH", config_file.path())
+        .env("FIL_PROOFS_PARAMETER_CACHE", tmp_dir.path())
+        .arg("fetch-params")
+        .arg("--keys")
+        .assert()
+        .success();
+
+    let list_files = read_dir(tmp_dir.path()).unwrap();
+
+    assert!(list_files.count() > 0);
+}
+
+#[test]
+fn test_download_location_of_proof_parameter_files_default() {
+    let tmp_dir = TempDir::new().unwrap();
+    let mut tmp_dir_path_buf = tmp_dir.path().to_path_buf();
+    let config = Config {
+        client: Client {
+            data_dir: tmp_dir_path_buf.clone(),
+            ..Client::default()
+        },
+        ..Config::default()
+    };
+
+    let mut config_file = tempfile::Builder::new().tempfile().unwrap();
+    config_file
+        .write_all(toml::to_string(&config).unwrap().as_bytes())
+        .expect("Failed writing configuration!");
+
+    Command::cargo_bin("forest")
+        .unwrap()
+        .env("FOREST_CONFIG_PATH", config_file.path())
+        .arg("fetch-params")
+        .arg("--keys")
+        .assert()
+        .success();
+
+    tmp_dir_path_buf.push("filecoin-proof-parameters");
```

Comment:
```suggestion
```

Reply:
I don't understand this

Reply:
Change the code such that `tmp_dir_path_buf` is not mutable.